src/cljs/nr/account.cljs

```diff
@@ -49,6 +49,7 @@
   (swap! app-state assoc-in [:options :log-width] (:log-width @s))
   (swap! app-state assoc-in [:options :log-top] (:log-top @s))
   (swap! app-state assoc-in [:options :blocked-users] (:blocked-users @s))
+  (swap! app-state assoc-in [:options :emotes] (:emotes @s))
```

Comment:
I _hate_ this whole system, have I said this before? Lmao I don't want you to change it here, just voicing my displeasure instead of bottling it up so we can start brainstorming a better way.